data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetHealthTest.java

```diff
@@ -111,6 +113,18 @@ public void shouldReturnUnavailableWhenStoreNotAvailable() throws Exception {
     assertThat(request.getResponseCode()).isEqualTo(SC_SERVICE_UNAVAILABLE);
   }
 
+  @Test
+  public void shouldThrowBadRequestWhenInvalidSyncingStatus() {
+    when(chainDataProvider.isStoreAvailable()).thenReturn(true);
+    when(syncService.getCurrentSyncState()).thenReturn(SyncState.SYNCING);
+    final String syncingStatus = String.valueOf(dataStructureUtil.randomPositiveInt(99));
+    request.setOptionalQueryParameter(SYNCING_STATUS, syncingStatus);
+
+    assertThatThrownBy(() -> handler.handleRequest(request))
+        .isInstanceOf(BadRequestException.class)
+        .hasMessageContaining("Invalid syncing status code");
+  }
+
```

Comment:
NIT: We could have a test for the lower boundary (99) and a test for the upper boundary (599)